src/noit_check_tools.h

```diff
@@ -151,7 +151,7 @@ MTEV_HOOK_PROTO(check_postflight,
 
 #define BAIL_ON_RUNNING_CHECK(check) do { \
   if(!noit_should_run_check(check, NULL)) { \
```

Comment:
static mtev_log_stream_t debug_cluster= NULL; \
if(!debug_cluster) debug_cluster = mtev_log_stream_find("debug/cluster"); \
mtevL(debug_cluster, "Check %s is running on another node\n", check->name); \